controls/radslider/features/tooltip.md

```diff
@@ -0,0 +1,57 @@
+---
+title: ToolTip
+page_title: ToolTip
+description: ToolTip
+slug: radslider-tooltip
+tags: tooltip
+published: True
+position: 8
+---
+
+# ToolTip
+
+The RadSlider control supports tool tip functionality. This feature allows the user to see the value of the slider's thumb while dragging it. In order to enable the ToolTip in the RadSlider component, you can set the __AutoToolTipPlacement__ property. The property is an enumeration of type AutoToolTipPlacement and the possible values which can be set are __None__, __TopLeft__, __BottomRight__. The default value of the __AutoToolTipPlacement__ property is __None__.
+
+#### __[XAML] Example 1: Set AutoToolTipPlacement property to TopLeft in horizontal Orientation__
```

Comment:
Is there a difference in setting the AutoToolTipPlacement property depending on the orientation? If no, probably the "in horizontal orientation" part is not necessary.